PayPal/src/test/java/com/braintreepayments/api/PayPalClientUnitTest.java

```diff
@@ -717,7 +717,7 @@ public void onBrowserSwitchResult_whenCancelUriReceived_notifiesCancellationAndS
         assertEquals("User canceled PayPal.", exception.getMessage());
         assertTrue(((UserCanceledException) exception).isExplicitCancelation());
 
-        verify(braintreeClient).sendAnalyticsEvent(eq("paypal.single-payment.browser-switch.canceled"));
+        verify(braintreeClient).sendAnalyticsEvent("paypal.single-payment.browser-switch.canceled", null);
```

Comment:
I would expect that we have an `payPalContextId` in these canceled cases.

Reply:
I added couple of tests here [6cdee58](https://github.com/braintree/braintree_android/pull/934/commits/6cdee584961d178e90d2c95da5ce930441b41335)